_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+
+Luckily, there is a workaround for that: lease takeovers. If you can find a suitable (i.e. creditworthy) person who is willing to take over your lease at an earlier date, then you will be released from your obligation to honour the contract until its term. In order for that to work, that tenant needs to actually sign the lease. The landlord is free to refuse the new tenant you propose, but if they are unable to provide a legally valid reason (e.g. “this person’s salary is too low for the rent”), then you will be released from the contract. They may try to intimidate you into staying in the apartment until the end of the term by saying that they will refuse anyone you propose, but that is not legal.
+
+## Rent increases
+While you stay in an apartment, your rent cannot change arbitrarily. However, under specific circumstances, it may come to change. Rent variations throughout a lease are indexed on the reference mortgage rate[^mortgage] and the  consumer price index (IPC)[^IPC]. The former keeps track of the average mortgage rent across banks in the country at a given time. It is updated every 3 months, and is clipped to the nearest multiple of 0.25%. One can check its evolution [here](https://www.bwo.admin.ch/bwo/fr/home/mietrecht/referenzzinssatz/entwicklung-referenzzinssatz-und-durchschnittszinssatz.html). The latter is a measure of inflation.
+
+Whenever these indices change, your rent can change. In practice, the IPC increases If they increase, it allows your landlord to increase your rent. Conversely, if they decrease, you are entitled to request a decrease in the rent. In either case, the new rent will be effective at the next term of your contract (usually your move-in date).
+
+:warn: In case of a rent decrease, it is your own responsibility to request the rent decrease, the landlord will not do it for you. You should always request it if applicable. It not only affects your rent, but the one the next tenants will pay. 
+
+### Calculation
+If you receive a notice of rent increase, it is important that you check that it is in accordance with the law. A proper notice must follow a certain format[^increase_format], lest it is deemed invalid. The increase should be broken down into 3 categories:
+- Increase of the reference mortgage rate: in the current conditions, this induces a 3% increase per 0.25% of increase in the mortgage rate[^mortgage_increase]. $\Delta R = \frac{\text{new rate} - \text{old rate}}{0.25} \times 0.03 \times R$
+- Increase of the IPC: 40% of the relative increase of the index can be applied to your rent: $\Delta R = \frac{\text{new index} - \text{old index}}{\text{old index}} \times 0.4 \times R$
```

Comment:
Unfortunately I don't think we have a LaTeX-HTML thing on the website :cry: Can you rewrite this using just Unicode characters? Like × and Δ. I use [symbl.cc](https://symbl.cc/) to browse characters.